core/trino-spi/src/main/java/io/trino/spi/function/FunctionBundle.java

```diff
@@ -11,18 +11,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.trino.metadata;
-
-import io.trino.spi.function.AggregationFunctionMetadata;
-import io.trino.spi.function.AggregationImplementation;
-import io.trino.spi.function.BoundSignature;
-import io.trino.spi.function.FunctionDependencies;
-import io.trino.spi.function.FunctionDependencyDeclaration;
-import io.trino.spi.function.FunctionId;
-import io.trino.spi.function.FunctionMetadata;
-import io.trino.spi.function.InvocationConvention;
-import io.trino.spi.function.ScalarFunctionImplementation;
-import io.trino.spi.function.WindowFunctionSupplier;
+package io.trino.spi.function;
```

Comment:
cc @dain

Reply:
I don't need to move FunctionBundle to spi after your suggestions